src/simularium/VisDataParse.ts

```diff
@@ -1,119 +1,108 @@
-import {
-    FrameData,
-    VisDataMessage,
-    AgentData,
-    AGENT_OBJECT_KEYS,
-} from "./types";
+import { VisDataMessage, AGENT_OBJECT_KEYS, CachedFrame } from "./types";
```

Comment:
is this now the only module that needs to use AGENT_OBJECT_KEYS?

Reply:
No it's also used in `getAgentDataFromBuffer`

I think you are right that things could get broken if `AGENT_OBJECT_KEYS` and the shape of `AgentData` got out of sync. Deriving one from the other would take a little bit of fussy typing probably but be doable.  I don't entirely follow the suggestion to make it part of Agent (VisAgent?) class though